db/rdbms/src/main/resources/mapper/ProcessDefinitionMapper.xml

```diff
@@ -320,4 +320,147 @@
     VALUES (#{processDefinitionKey}, #{processDefinitionId}, #{resourceName}, #{bpmnXml}, #{name},
             #{tenantId}, #{versionTag}, #{version}, #{formId})
   </insert>
+
+  <select id="processInstanceStatisticsCount" parameterType="io.camunda.db.rdbms.read.domain.ProcessDefinitionInstanceStatisticsDbQuery"
+    resultType="long">
+    SELECT COUNT(*)
+    FROM (
+      SELECT pi.PROCESS_DEFINITION_ID
+      FROM ${prefix}PROCESS_INSTANCE pi
+      <where>
+        pi.STATE = 'ACTIVE'
+        <if test="authorizedResourceIds != null and !authorizedResourceIds.isEmpty()">
+          AND pi.PROCESS_DEFINITION_ID IN
+          <foreach collection="authorizedResourceIds" item="resourceId" open="(" separator="," close=")">
+            #{resourceId}
+          </foreach>
+        </if>
+        <if test="authorizedTenantIds != null and !authorizedTenantIds.isEmpty()">
+          AND pi.TENANT_ID IN
+          <foreach collection="authorizedTenantIds" item="tenantId" open="(" separator="," close=")">
+            #{tenantId}
+          </foreach>
+        </if>
+      </where>
+      GROUP BY pi.PROCESS_DEFINITION_ID, pi.TENANT_ID
+    )
+  </select>
+
+  <select id="processInstanceStatistics" parameterType="io.camunda.db.rdbms.read.domain.ProcessDefinitionInstanceStatisticsDbQuery"
+    resultType="io.camunda.search.entities.ProcessDefinitionInstanceStatisticsEntity">
+    SELECT pi.PROCESS_DEFINITION_ID,
+    pi.TENANT_ID,
+    MAX(pd.NAME) AS LATEST_PROCESS_DEFINITION_NAME,
```

Comment:
💭 Interesting. Ideally, this is the name of the latest version, not the maximum of the names, right? 🤔